lib/v2/local.js

```diff
@@ -120,7 +123,7 @@ var dorita980 = function localV2 (user, password, host, emitIntervalTime) {
     getWirelessStatus: () => waitPreferences(false, ['wifistat', 'netinfo'], true),
     getCloudConfig: () => waitPreferences(false, ['cloudEnv'], true),
     getSKU: () => waitPreferences(false, ['sku'], true),
-    start: () => _apiCall('cmd', 'start'),
+    start: (args) => _apiCall('cmd', 'start', args),
```

Comment:
I would like to add a new method instead of enance the start() method. something like:
`cleanRoom: (args) => _apiCall('cmd', 'start', args),`